screens/Home.js

```diff
@@ -42,8 +42,8 @@ const Home = ({ navigation }) => {
     const searchFilter = (text) => {
         if (text) {
             const newData = data.filter((item) => {
-                const itemData = item.name ? unorm.nfd(item.name).replace(/[\u0300-\u036f]/g, '').toUpperCase() : ''.toUpperCase();
-                const textData = unorm.nfd(text).replace(/[\u0300-\u036f]/g, '').toUpperCase();
+                const itemData = item.name ? nomarlizeAndUpper(item.name) : ''.toUpperCase();

```

Comment:
You could use `??` operator instead of this one. Because `''.toUpperCase()` is unnecessary